src/main/java/games/strategy/engine/framework/ui/NewGameChooserEntry.java

```diff
@@ -37,9 +39,21 @@ private String getLowerCaseComparable(final NewGameChooserEntry newGameChooserEn
     };
   }
 
+  public NewGameChooserEntry() {
+    this(defaultGameUri());
+  }
+
+
+  private static URI defaultGameUri() {
+    final String defaultLocation = ClientSetting.SELECTED_GAME_LOCATION.value();
+    try {
+      return new URI(defaultLocation);
+    } catch (final URISyntaxException e) {
+      throw new IllegalStateException("Illegal default game location: " + defaultLocation, e);
+    }
+  }
 
-  public NewGameChooserEntry(final URI uri)
-      throws IOException, GameParseException, SAXException, EngineVersionException {
+  public NewGameChooserEntry(final URI uri) {
```

Comment:
This change causes my static analyzer to flag the following warnings about `NewGameChooserModel#createEntry()`:

- The declared exception GameParseException is not actually thrown by the method createEntry(URI) from type NewGameChooserModel
- The declared exception IOException is not actually thrown by the method createEntry(URI) from type  NewGameChooserModel
- The declared exception SAXException is not actually thrown by the method createEntry(URI) from type NewGameChooserModel
- The declared exception EngineVersionException is not actually thrown by the method createEntry(URI) from type NewGameChooserModel